public/content/documentation/web/component/link.md

```diff
@@ -6,75 +6,485 @@ How to test a link
+- A link can look like a big shiny button but it must be coded as `<a>` link
 
+### If it does something, it's a `<button>`
 
-Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/link)
+- Buttons cause an action to occur on the same page
+  - Submit a form (even when submission takes you to a new page)
+  - Open a menu
+  - Launch a modal
+  - Expand details
+- A button can look like a link, but it must be coded as a `<button>`
+
+## Code examples
+
+### Use semantic HTML with common sense names
+
+This semantic HTML contains all accessibility features by default. 
+
+```html
+<a href="/about/">
+  About
+</a>
+```
+
+<example>
+<a href="/about/">
+  About
+</a>
+</example>
+
+### Name links logically
+
+- **Do not** use a heading with a generic link below. 
+- Instead, make the heading a link or programmatically associate the link with the heading using <code>aria-describedby</code>.
+
+#### Bad example
+
+```html
+<h3>About our coffee subscriptions</h3>
+<p>Get the best coffee delivered to your door</p>
+<a href="/about/">
+   Learn more
+</div>
+```
 
-## Developer Notes
+#### Good examples
 
-### Name
+```html
+<h3><a href="/about/">About our coffee subscriptions</a></h3>
+<p>Get the best coffee delivered to your door</p>
+```
 
-- Typically doesn’t have a name or description since there must be only one instance per page.
+```html
+<h3 id="unique-id">About our coffee subscriptions</h3>
+<p>Get the best coffee delivered to your door</p>
+<a href="/about/" aria-describedby="unique-id">
+   Learn more
+</div>
+```
 
-## Videos
+### Making a link with no `href` focusable
 
-- Videos go here
-<video controls>
-  <source src="media/video/native/button/buttonIosVoiceover.webm" type="video/webm">
-  Your browser does not support the video tag.
-</video>
+- **Do not** put anything but a URI in the `href`
+- A link with no `href` will not be focusable with the keyboard without `tabindex="0"`.
+- Add `role="link"` to ensure screen reader reads the role
+
+```html
+<a tabindex="0" role="link">
+  About
+</a>
+```
+
+<example>
+<a tabindex="0" role="link">
+  About
+</a>
+</example>
+
+### Avoid custom elements
+
+This custom button requires extra attributes and keyboard event listeners.
+
+```html
+<custom-element role="link" tabindex="0">
+  About
+</custom-element>
+```
+
+### Repeating text links
+
+Sometimes the design will call for multiple links with the same text label. In a case like this, `aria-label` can be used to name each link's purpose.
+
+```html
+<button>Get free coffee</button>
+<a href="/free-coffee-tc/" aria-label="Free coffee terms and conditions">
+  Terms & Conditions
+</a>
+<button>Get free donuts</button>
+<a href="/free-donuts-tc/" aria-label="Free donuts terms and conditions">
+  Terms & Conditions
+</a>
+```
+
+### Don't duplicate the visible text name in the aria-label
```

Comment:
Elsewhere on the page, we refer to 'aria-label' in code ticks, let's do that here for consistency.